scripts/ovs/templates/small/setup.json

```diff
@@ -0,0 +1,711 @@
+        "domains": {"domain_guids": []},
+        "scaling": "LOCAL",
+        "presets": [ ],
+        "osds": {
+          "{{storage_01_ip}}": {
+            "{{storage_01_disk_1}}": 1,
+            "{{storage_01_disk_2}}": 1,
+            "{{storage_01_disk_3}}": 1,
+            "{{storage_01_disk_4}}": 1,
+            "{{storage_01_disk_5}}": 1,
+            "{{storage_01_disk_6}}": 1,
+            "{{storage_01_disk_7}}": 1,
+            "{{storage_01_disk_8}}": 1,
+            "{{storage_01_disk_9}}": 1,
+            "{{storage_01_disk_10}}": 1,
+            "{{storage_01_disk_11}}": 1,
+            "{{storage_01_disk_12}}": 1,
+            "{{storage_01_disk_13}}": 1,
+            "{{storage_01_disk_14}}": 1,
+            "{{storage_01_disk_15}}": 1
+          },
+          "{{storage_02_ip}}": {
+            "{{storage_02_disk_1}}": 1,
+            "{{storage_02_disk_2}}": 1,
+            "{{storage_02_disk_3}}": 1,
+            "{{storage_02_disk_4}}": 1,
+            "{{storage_02_disk_5}}": 1,
+            "{{storage_02_disk_6}}": 1,
+            "{{storage_02_disk_7}}": 1,
+            "{{storage_02_disk_8}}": 1,
+            "{{storage_02_disk_9}}": 1,
+            "{{storage_02_disk_10}}": 1,
+            "{{storage_02_disk_11}}": 1,
+            "{{storage_02_disk_12}}": 1,
+            "{{storage_02_disk_13}}": 1,
+            "{{storage_02_disk_14}}": 1,
+            "{{storage_02_disk_15}}": 1
+          },
+          "{{storage_03_ip}}": {
+            "{{storage_03_disk_1}}": 1,
+            "{{storage_03_disk_2}}": 1,
+            "{{storage_03_disk_3}}": 1,
+            "{{storage_03_disk_4}}": 1,
+            "{{storage_03_disk_5}}": 1,
+            "{{storage_03_disk_6}}": 1,
+            "{{storage_03_disk_7}}": 1,
+            "{{storage_03_disk_8}}": 1,
+            "{{storage_03_disk_9}}": 1,
+            "{{storage_03_disk_10}}": 1,
+            "{{storage_03_disk_11}}": 1,
+            "{{storage_03_disk_12}}": 1,
+            "{{storage_03_disk_13}}": 1,
+            "{{storage_03_disk_14}}": 1,
+            "{{storage_03_disk_15}}": 1
+          },
+          "{{storage_04_ip}}": {
+            "{{storage_04_disk_1}}": 1,
+            "{{storage_04_disk_2}}": 1,
+            "{{storage_04_disk_3}}": 1,
+            "{{storage_04_disk_4}}": 1,
+            "{{storage_04_disk_5}}": 1,
+            "{{storage_04_disk_6}}": 1,
+            "{{storage_04_disk_7}}": 1,
+            "{{storage_04_disk_8}}": 1,
+            "{{storage_04_disk_9}}": 1,
+            "{{storage_04_disk_10}}": 1,
+            "{{storage_04_disk_11}}": 1,
+            "{{storage_04_disk_12}}": 1,
+            "{{storage_04_disk_13}}": 1,
+            "{{storage_04_disk_14}}": 1,
+            "{{storage_04_disk_15}}": 1
+            }
+        }
+      },
+      {
+        "name": "backend02",
+        "domains": {"domain_guids": []},
+        "scaling": "LOCAL",
+        "presets": [ ],
+        "osds": {
+          "{{storage_01_ip}}": {
```

Comment:
this should be more dynamic loop over storage nodes and use hdd / 2 disks (2nd half)